### PR TITLE
Fixed #2124 - SuiteP dashlets issues with icons + broken icons in das…

### LINKED ISSUE #2124: SuiteP dashlets issues with icons + broken icons in dashlets after uprade from 7.6.x versions
#### Issue

- The texts for Website + News Feed dashlet is missing in dashled window 
- The icons are missing (or are old)  in dahslet pop up window after upgrade
- Broken icons of existing dashlets after upgrade
#### Steps to Reproduce

1. Home page
2. Action menu button
3. Add Dashlet
4. Web Tab in Dashlet pop up window

dashlet1
#### Context

When we upgraded from 7.6.x to 7.7.x , we see those broken icons for web dashlet / jotpad /custom modules icons 

dashlet2
- i have not change icons of dashlets after upgrade

dashlet3

dashlet4

dashlet5
#### Your Environment

- SuiteCRM Version used:
- Browser name and version Chrome Version 53.0.2704.63 (64-bit)
- Environment name and version MySQL 5.6, PHP 5.6:
- Operating System and version : Hosting -- Linux

### PR DESCRIPTION
**Pull request by shogunpol**
Fixed - SuiteP dashlets issues with icons
## Description

Fixed - SuiteP dashlets issues with icons
References issue #2124 

## Motivation and Context

Fix apearience of SuiteP issues.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

…hlets after uprade from 7.6.x versions

### CODE REVIEW

File: themes/SuiteP/include/MySugar/tpls/addDashletsDialog.tpl [hunk 1]
```diff
@@ -147,7 +147,7 @@
 	<table width="95%">
         <tr>
 	        <td scope="row"></td>
-	        <td><input type="text" id="rss_address" value="http://" style="width: 400px"  title="{sugar_translate label='LBL_RSS_TITLE' module='Home'}" /></td>
```
Comment (ijdavie) [inline themes/SuiteP/include/MySugar/tpls/addDashletsDialog.tpl]: This translate label should be in the previous td.

File: themes/SuiteP/include/MySugar/tpls/addDashletsDialog.tpl [hunk 2]
```diff
@@ -134,7 +134,7 @@
 	<table width="95%">
 	    <tr>
 	        <td scope="row"></td>
-	        <td><input type="text" id="web_address" value="http://" style="width: 400px"   title="{sugar_translate label='LBL_WEBSITE_TITLE' module='Home'}"/></td>
```
Comment (ijdavie) [inline themes/SuiteP/include/MySugar/tpls/addDashletsDialog.tpl]: This Translate label should be in the previous td.

File: themes/SuiteP/include/MySugar/tpls/addDashletsDialog.tpl [hunk 3]
```diff
@@ -147,7 +147,7 @@
 	<table width="95%">
         <tr>
 	        <td scope="row"></td>
-	        <td><input type="text" id="rss_address" value="http://" style="width: 400px"  title="{sugar_translate label='LBL_RSS_TITLE' module='Home'}" /></td>
+	        <td>{sugar_translate label='LBL_RSS_TITLE' module='Home'}<br /><input type="text" id="rss_address" value="http://" style="width: 400px"  title="{sugar_translate label='LBL_RSS_TITLE' module='Home'}" /></td>
```
Comment (ijdavie) [inline themes/SuiteP/include/MySugar/tpls/addDashletsDialog.tpl]: This Translate label should be in the previous td.

**Comment by shogunpol** [comment]
@ijdavie  done

File: config_override.php [hunk 4]
```diff
@@ -1,3 +1,10 @@
 <?php
 /***CONFIGURATOR***/
+$sugar_config['default_export_charset'] = 'ISO-8859-1';
```
Comment (ijdavie) [inline config_override.php]: config_override.php should not be in this pull request.

**Pull request closed without merging**